src/components/ItaliaTheme/View/Commons/SideMenu.jsx

```diff
@@ -171,11 +176,7 @@ const SideMenu = ({ data, content_uid }) => {
                     role="progressbar"
                   />
                 </div>
-                <AccordionBody
-                  active={isNavOpen}
-                  id="side-menu-body"
-                  role="region"
-                >
+                <AccordionBody active={isNavOpen} id="side-menu-body">
```

Comment:
perchè hai tolto il role="region"? 
hai verificato che continua a funzionare bene il lettore di schermo in questa parte e la navigazione da tastiera?

Reply:
Si, ho verificato e confermo che funziona come prima. Il role="region" viene usato per identificare una regione importante al sito, forse qualcuno aveva pensato che fosse importante dare stacco al menu, ma questo il nav e l'aside lo fanno in automatico senza creare una regione in più dentro al menu laterale.

<img width="832" alt="Screenshot 2025-01-30 alle 15 13 57" src="https://github.com/user-attachments/assets/18fa2e4a-cff9-440c-99f2-ac9ccecdca8c" />